### PR TITLE
scroll for typeahead

### PR DESCRIPTION
**Pull request by arins**
#1168 Fixed scroll for typeahead.

### CODE REVIEW

File: components/position.ts [hunk 1]
```diff
@@ -126,6 +127,14 @@ export class PositionService {
     return (nativeEl.style as KeyAttribute)[cssProp];
   }
 
+  private get window():Window {
```
Comment (valorkin) [inline components/position.ts:130]: this one can create issues with universal >.<
use https://github.com/valor-software/ng2-bootstrap/blob/development/components/utils/facade/browser.ts

File: components/typeahead/typeahead-container.component.ts [hunk 2]
```diff
@@ -8,20 +8,20 @@ import { TypeaheadDirective } from './typeahead.directive';
 import { TypeaheadMatch } from './typeahead-match.class';
 
 const bs4 = `
-  <div class="dropdown-menu"
-       [ngStyle]="{top: top, left: left, display: 'block'}"
+  <div #ulElement class="dropdown-menu"
```
Comment (valorkin) [inline components/typeahead/typeahead-container.component.ts:11]: html ids in reusable component? - no way
it will be a mess

**Review by valorkin — COMMENTED** [review]
in general looks, good
but couple issue need to be fixed

Huge thanks for your time!

File: components/typeahead/typeahead-container.component.ts (hunk 2 above)
Comment (arins) [reply components/typeahead/typeahead-container.component.ts:11]: I will make a component of each dom element we want to grab. So we don't need to use element referencing by hashtag. 
When the LI and UL element are angular 2 components I can just use the code below in typehead-container.component

`@ViewChild(TypeaheadListComponent) list: TypeaheadListComponent`
`@ViewChildren(TypeaheadListItemComponent) listItems: QueryList<TypeaheadListItemComponent>`

So I will create additional components just for the UL element (`TypeaheadListComponent`)  and a component for the LI elements (`TypeaheadListItemComponent`). This way I can grab the dom elements without hashtag. What do you think about that?

File: components/position.ts (hunk 1 above)
Comment (arins) [reply components/position.ts:130]: Sorry I misunderstood you. How do you mean that it creates issues? Should I not use position.ts?

File: components/typeahead/typeahead-container.component.ts (hunk 2 above)
Comment (valorkin) [reply components/typeahead/typeahead-container.component.ts:11]: sorry sorry, my mind interpreted ng2 ids as html ids,
forgive me ;)
forget about this one

File: components/position.ts (hunk 1 above)
Comment (arins) [reply components/position.ts:130]: Is this something I need to fix? Should I not use position.ts?

**Pull request closed without merging**